composer.json

```diff
@@ -21,15 +21,16 @@
         "source": "https://github.com/xiidea/EasyAuditBundle"
     },
     "require": {
-        "php": "^7.2.5",
+        "php": "^8.0.0",
         "psr/log": "^1",
         "symfony/framework-bundle": ">=5.0 <6.0.0",
-        "symfony/security-bundle": ">=5.0 <6.0.0"
+        "symfony/security-bundle": ">=5.0 <6.0.0",
+        "symfony/symfony": "5.2.*"
```

Comment:
Should not add symfony as dependency. The library does not depends on symfony directly.